docs/designers-developers/developers/tutorials/sidebar-tutorial/plugin-sidebar-0.md

```diff
@@ -1,12 +0,0 @@
-# Creating a Sidebar for Your Plugin
```

Comment:
Moved to `docs/tutorials/sidebar-tutorial/plugin-sidebar-0.md`. It looks like GitHub didn't pick up this rename.